frontend/src/api/Socket.ts

```diff
@@ -32,12 +29,11 @@ export const isValidNickname = (nickname: string) => nickname.length > 0
  * @returns void Promise, reject if socket is already connected
  * or fails to connect.
 */
-export const connect = (roomId: string, userId: string):
+export const connect = (userId: string):
   Promise<void> => new Promise<void>((resolve, reject) => {
     if (!stompClient || !stompClient.connected) {
       // Connect to given endpoint, subscribe to future messages, and send user message.
-      socketRoomId = roomId;
-      const socket: WebSocket = new SockJS(routes(socketRoomId).connect);
+      const socket: WebSocket = new SockJS(routes().connect);
```

Comment:
Based on looking at the code before, the `socketRoomId` variable was never used in any way since it is not a component in the `connect` URL, and wasn't used anywhere else. Thus, I just removed that, which means I could make the `roomId` optional.

I'm not sure how it would work anyway if there were multiple rooms simultaneously anyway - on the same tab? I don't think that worked before, and I'm not sure how this would affect that - lmk if I'm missing something here.

Reply:
I see, thanks - yeah I think I was tripped up about roomId being optional for the paths that require it, but I see that the only time when roomId isn't provided is for the connect endpoint, so that all makes sense